ggml-sycl.cpp

```diff
@@ -12166,9 +12185,13 @@ inline void ggml_sycl_op_dequantize_mul_mat_vec(
             src1_dfloat = (sycl::half *)src1->data + src1_padded_row_size;
         } else {
             src1_dfloat = src1_dfloat_a.alloc(ne00);
+            //ggml_cpy_f32_f16_sycl((const char *)src1_ddf_i, (char *)src1_dfloat,
+            //                      ne00, ne00, 1, sizeof(float), 0, 0, ne00, 1,
```

Comment:
I would argue it is worth deleting this old code. 
We can find it again using the git history if we need to roll back this change.